packages/ui-focusable/src/Focusable/__tests__/Focusable.test.tsx

```diff
@@ -331,32 +327,31 @@ describe('<Focusable />', async () => {
   })
 
   it('should provide a focus method', async () => {
-    let focusable
+    let focusable: Focusable
     await mount(
       <Focusable
-        // @ts-expect-error ts-migrate(2769) FIXME: No overload matches this call.
-        componentRef={(el) => {
+        //@ts-expect-error TODO this is coming from ReactComponentWrapper
+        componentRef={(el: Focusable) => {
```

Comment:
this prop is added only during testing by ReactComponentWrapper, I had no good idea who to type it.